NAMESPACE

```diff
@@ -137,6 +137,7 @@ export(use_github_release)
 export(use_gitlab_ci)
 export(use_gpl3_license)
 export(use_gpl_license)
+export(use_import_from)
```

Comment:
Not sure where I fall on this, but it seem we should also consider `use_importfrom()`